pkg/processor/runtime/python/runtime.go

```diff
@@ -214,10 +218,19 @@ func (py *python) handleEvent(functionLogger nuclio.Logger, event nuclio.Event,
 				return
 			}
 
+			switch unmarshalledResult.BodyEncoding {
+			case "":
```

Comment:
I don't think it's very nice to have an empty string here. 
What's the equivalent here? It could either be `text`, or even `none`- but it has to hold a value

Reply:
Done